.goreleaser.yaml

```diff
@@ -8,19 +9,24 @@ builds:
       - windows
       - darwin
 archives:
-  - replacements:
-      386: i386
-      amd64: x86_64
+  - name_template: >-
```

Comment:
Does this result in the same file names that we currently have? I'm thinking about existing dockerfiles that download the tool.

Reply:
Yep. I tested locally and it produces the same files as before. For the current snapshot:

```
yardl_0.6.4-next_darwin_arm64.tar.gz
yardl_0.6.4-next_darwin_x86_64.tar.gz
yardl_0.6.4-next_linux_arm64.tar.gz
yardl_0.6.4-next_linux_i386.tar.gz
yardl_0.6.4-next_linux_x86_64.tar.gz
yardl_0.6.4-next_windows_arm64.zip
yardl_0.6.4-next_windows_i386.zip
yardl_0.6.4-next_windows_x86_64.zip
```

The required change to the config is documented [here](https://goreleaser.com/deprecations/#archivesreplacements) and I adjusted the `name_template` slightly to match exactly what we had before.